test/grpc/BUILD.bazel

```
@@ -53,6 +53,36 @@ cc_binary(
     }),
 )
 
+cc_library(
+    name = "grpc_streaming",
+    srcs = [
+        "streaming/streaming.h",
+        "streaming/write_last_after_reply_two_requests.cc",
+        "streaming/write_last_before_reply_one_request.cc",
+        "streaming/write_last_before_reply_two_requests.cc",
+        "streaming/writes_done_after_reply_one_request.cc",
+        "streaming/writes_done_after_reply_two_requests.cc",
+        "streaming/writes_done_before_reply_one_request.cc",
+        "streaming/writes_done_before_reply_two_requests.cc",
+        "test.h",
+    ],
+    copts = copts(),
+    target_compatible_with = select({
```

Please copy any associated comments when you copy code so that we know why we're doing things in particular ways!
```suggestion
    # TODO(benh): resolve build issues on Windows and then remove
    # these 'target_compatible_with' constraints.
    target_compatible_with = select({
```